src/components/CohortBuilder/Summary/SurvivalChart.js

```diff
@@ -262,6 +264,15 @@ class SurvivalChart extends React.Component {
         y: event.offsetY,
       },
     }));
+
+    if (this.props.analyticsTracking) {
+      const label = { donor: donors[0] };
```

Comment:
grab all the whole donors array not just the first index

Reply:
Changed to:
```javascript
const label = { donors };
```